docs/content/preview/architecture/transactions/read-committed.md

```diff
@@ -1564,4 +1567,4 @@ If a statement in the Read Committed isolation level faces a conflict, it will b
 
 You can set these parameters on a per-session basis, or in the `ysql_pg_conf_csv` YB-TServer gflag on cluster startup.
 
-After the optimized version of pessimistic locking (as described in [Cross-feature interaction](#cross-feature-interaction)) is completed, there won't be a need to manually tune these parameters for performance. Statements will restart only when all conflicting transactions have committed or rolled back, instead of retrying with an exponential backoff.
+If the [Wait-on-Conflict](../concurrency-control/#wait-on-conflict) concurrency control policy is enabled, there won't be a need to manually tune these parameters for performance. Statements will restart only when all conflicting transactions have committed or rolled back, instead of retrying with an exponential backoff.
```

Comment:
I manually added an anchor link in concurrency-control.md so this link works right. It will continue to work right after we eventually remove the beta tag from that heading in concurrency-control.md, too.